langkit/toxicity.py

```diff
@@ -4,6 +4,11 @@
 from whylogs.experimental.core.udf_schema import register_dataset_udf
 from langkit import LangKitConfig, lang_config, prompt_column, response_column
 
+import os
+import torch
+
+_USE_CUDA = torch.cuda.is_available() and not bool(os.environ.get("WHYLABS_DISABLE_CUDA", False))
```

Comment:
It’s not disabling CUDA, suggest rename the env var to indicate it’s disabling running LangKit on CUDA?